app/src/main/java/com/example/narutoquiz/ui/mainScreen/main/ErrorDialogFragment.kt

```diff
@@ -0,0 +1,30 @@
+package com.example.narutoquiz.ui.mainScreen.main
+
+import android.app.AlertDialog
+import android.app.Dialog
+import android.os.Bundle
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.DialogFragment
+import com.example.narutoquiz.R
+import com.example.narutoquiz.databinding.DialogGameBinding
+
+class ErrorDialogFragment(
+    private val mainScreen: () -> Unit,
+) : DialogFragment() {
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        val binding: DialogGameBinding = DataBindingUtil.inflate(
+            this.requireActivity().layoutInflater, R.layout.dialog_error, null, false
```

Comment:
java.lang.ClassCastException: com.example.narutoquiz.databinding.DialogErrorBindingImpl cannot be cast to com.example.narutoquiz.databinding.DialogGameBinding